bindata/manifests/operator-webhook/003-webhook.yaml

```diff
@@ -14,6 +18,9 @@ webhooks:
         name: operator-webhook-service
         namespace: {{.Namespace}}
         path: "/mutating-custom-resource"
+      {{if .CaBundle}}
```

Comment:
Could we check the "ClusterType" to be equals to "kubernetes" as well? Just in case `WEBHOOK_CA_BUNDLE` is accidentally set in "openshift" environment.

Reply:
This would apply to other manifests in bindata.

Reply:
The idea I had here is that passing the `CaBundle` would also be an option in Openshift. Thinking on situations where one would have a quick test or dev cluster where the ca operator is not deployed. This might not make sense though as I am not familiar with Openshift. If it does not, I can add the check for cluster type there as you mention. Let me know.

Reply:
@jcaamano thanks for all the consideration!
We didn't do ca injection manually in openshift afaik, I think we could add it only for k8s at this moment.
If anything changes, we could remove the conditional check.

Reply:
@zshi-redhat I rewrote the conditions so that `CaBundle` is only considered for `ClusterType=kubernetes`. With `ClusterType=openshift`, `CaBundle` is ignored.